app/controllers/question_answer/votes_controller.rb

```diff
@@ -4,47 +4,47 @@ module QuestionAnswer
   class VotesController < ::ApplicationController
     before_action :ensure_logged_in
     before_action :find_vote_post
-    before_action :find_vote_user, only: [:create, :destroy]
```

Comment:
Removed because I don't think we want to allow users to vote on behalf of another user. Always use the current user.